packages/beacon-node/src/network/core/metrics.ts

```diff
@@ -214,12 +213,12 @@ export function createNetworkCoreMetrics(register: RegistryMetricCreator) {
         name: "lodestar_attnets_service_long_lived_subscriptions_total",
         help: "Count of long lived subscriptions",
       }),
-      subscribeSubnets: register.gauge<{subnet: number; src: SubnetSource | DLLSubnetSource}>({
+      subscribeSubnets: register.gauge<{subnet: number; src: DLLSubnetSource}>({
         name: "lodestar_attnets_service_subscribe_subnets_total",
         help: "Count of subscribe_subnets calls",
         labelNames: ["subnet", "src"],
       }),
-      unsubscribeSubnets: register.gauge<{subnet: number; src: SubnetSource | DLLSubnetSource}>({
+      unsubscribeSubnets: register.gauge<{subnet: number; src: DLLSubnetSource}>({
```

Comment:
might wanna rename this now as there is only one type of subnet now

```suggestion
      unsubscribeSubnets: register.gauge<{subnet: number; src: SubnetSource}>({
```